cognite/client/__init__.py

```diff
@@ -1,3 +1,3 @@
 from cognite.client._cognite_client import CogniteClient
 
-__version__ = "2.5.4"
+__version__ = "2.5.5"
```

Comment:
We're on making a change to docs right? If so, we shouldn't bump

Reply:
That makes my work easier. Thanks!